Chat/src/app/theming/SwitchableFluentThemeProvider.tsx

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT license.
 
 import React, { useState, useMemo, createContext, useContext } from 'react';
-import { FluentThemeProvider, lightTheme, darkTheme } from '@azure/communication-react';
+import { FluentThemeProvider, lightTheme, darkTheme } from '@internal/react-components';
```

Comment:
Replace `@internal` references with `@azure/communication-react`

Upstream note.: Just like storybook, we should force the use of external name in samples.

Reply:
Done